package.json

```diff
@@ -58,7 +58,7 @@
     "serve": "serve --no-clipboard --single --listen 3000 build",
     "eject": "react-scripts eject",
     "lint": "eslint --fix --ext .js,.ts,.tsx ./src --ignore-path .gitignore",
-    "prettier": "prettier --ignore-path .gitignore --write \"**/*.+(js|json|ts|tsx)\"",
```

Comment:
wy does prettier should not write `.json` files ?

Reply:
yes, u're right, we should keep it